### PR TITLE
MAINTAINERS: add @mfrw to org

### PR DESCRIPTION
**Pull request by mfrw**
closes: #6142

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -48,6 +46,8 @@ If you are an owner of the organization, you can see an automated list
   [10 May 2021](https://github.com/tldr-pages/tldr/issues/5919) — present
 - **Seth Falco ([@SethFalco](https://github.com/SethFalco))**:
   [19 May 2021](https://github.com/tldr-pages/tldr/issues/5993) - present
+- **Muhammad Falak R Wani ([@mfrw](https://github.com/mfrw))**:
+  [6 September 2018](https://github.com/tldr-pages/tldr/issues/2306) — [21 June 2021](https://github.com/tldr-pages/tldr/issues/6142)
 - Owen Voke ([@owenvoke](https://github.com/owenvoke))
   [11 January 2018](https://github.com/tldr-pages/tldr/issues/1885) — [26 August 2018](https://github.com/tldr-pages/tldr/issues/2258)
```
Comment (navarroaxel) [inline MAINTAINERS.md:52]: I think this is the correct order:
```suggestion
- Owen Voke ([@owenvoke](https://github.com/owenvoke))
  [11 January 2018](https://github.com/tldr-pages/tldr/issues/1885) — [26 August 2018](https://github.com/tldr-pages/tldr/issues/2258)
- **Muhammad Falak R Wani ([@mfrw](https://github.com/mfrw))**:
  [6 September 2018](https://github.com/tldr-pages/tldr/issues/2306) — [21 June 2021](https://github.com/tldr-pages/tldr/issues/6142)
```

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
@mfrw I put you at the end of the collaborators list because we sort by when they were upgraded to org members, not when they became collaborators.

**Review by bl-ue — APPROVED** [review]
Almost 3 years! 😀

**Comment by mfrw** [comment]
> Almost 3 years! 😀

It took me a while to get there .. but.. better late than never :)

**Pull request merged**